SPECS/libarchive/libarchive.spec

```diff
@@ -76,7 +78,7 @@ make %{?_smp_mflags} check
 -   Fixed "Source0" and "URL" tags.
 -   License verified.
 *   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 3.3.3-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
+-   Initial CBL-Mariner import from Photon (license: dual Apache2/GPL2).
```

Comment:
Previous version was actually correct.

Reply:
Will update in a separate PR. good catch